x-pack/plugin/sql/src/main/java/org/elasticsearch/xpack/sql/querydsl/container/QueryContainer.java

```diff
@@ -81,23 +80,18 @@
 
     private final Set<Sort> sort;
     private final int limit;
+    private final boolean trackHits;
 
     // computed
     private Boolean aggsOnly;
     private Boolean customSort;
 
     public QueryContainer() {
-        this(null, null, null, null, null, null, null, -1);
+        this(null, null, null, null, null, null, null, -1, false);
     }
 
-    public QueryContainer(Query query,
-            Aggs aggs,
-            List<Tuple<FieldExtraction, ExpressionId>> fields,
-            AttributeMap<Attribute> aliases,
-            Map<String, GroupByKey> pseudoFunctions,
-            AttributeMap<Pipe> scalarFunctions,
-            Set<Sort> sort,
-            int limit) {
+    public QueryContainer(Query query, Aggs aggs, List<Tuple<FieldExtraction, ExpressionId>> fields, AttributeMap<Attribute> aliases,
```

Comment:
I don't know if it's the checkstyle that we follow, but personally I prefer the one arg per line for method signatures when the don't fit in one line, as it makes it more readable imho.

Reply:
Auto-formatting...